x/valset-pref/keeper_test.go

```diff
@@ -100,6 +103,63 @@ func (suite *KeeperTestSuite) AllocateRewards(ctx sdk.Context, delegator sdk.Acc
 	suite.Require().NotZero(rewardsAfterAllocation[0].Amount)
 }
 
+func (suite *KeeperTestSuite) SetupLocks(delegator sdk.AccAddress) []lockuptypes.PeriodLock {
+	locks := []lockuptypes.PeriodLock{}
+	// Setup lock
+	coinsToLock := sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 10_000_000)}
+	osmoToLock := sdk.Coins{sdk.NewInt64Coin(appParams.BaseCoinUnit, 10_000_000)}
+	suite.FundAcc(delegator, sdk.Coins{sdk.NewInt64Coin(sdk.DefaultBondDenom, 100_000_000), sdk.NewInt64Coin(appParams.BaseCoinUnit, 100_000_000)})
+
+	// happy lock case
+	TwoWeekDuration, _ := time.ParseDuration("336h")
```

Comment:
If we are nulling an error lets be sure to check it (idk if its an error or not)